Sources/Public/Animation/LottieView.swift

```diff
@@ -1,42 +1,112 @@
 // Created by Bryn Bodayle on 1/20/22.
 // Copyright © 2022 Airbnb Inc. All rights reserved.
 
+import Combine
 import SwiftUI
 
 // MARK: - LottieView
 
 /// A wrapper which exposes Lottie's `LottieAnimationView` to SwiftUI
 @available(iOS 13.0, tvOS 13.0, macOS 10.15, *)
-public struct LottieView: UIViewConfiguringSwiftUIView {
+public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
 
   // MARK: Lifecycle
 
   /// Creates a `LottieView` that displays the given animation
-  public init(animation: LottieAnimation?) {
-    self.animation = animation
+  public init(animation: LottieAnimation?) where Placeholder == EmptyView {
```

Comment:
Inits are still missing some documentation, will add it once we align on the implementation